web/app/frontend/src/types.tsx

```diff
@@ -0,0 +1,18 @@
+import { StackScreenProps } from '@react-navigation/stack';
```

Comment:
can we move this to the types folder?
and I think this doesn't need to be a tsx file, ts should be enough